wsmaster/che-core-api-workspace/src/main/java/org/eclipse/che/api/workspace/server/model/impl/WorkspaceImpl.java

```diff
@@ -90,6 +91,10 @@ public static WorkspaceImplBuilder builder() {
   @JoinColumn(name = "config_id")
   private WorkspaceConfigImpl config;
 
+  @OneToOne(cascade = CascadeType.ALL, orphanRemoval = true)
+  @JoinColumn(name = "devfile_id")
+  private DevfileImpl devfile;
```

Comment:
I think after adding devfile here, syncName must be updated

Reply:
Added